tensorflow_probability/python/mcmc/transformed_kernel.py

```diff
@@ -33,15 +33,11 @@
     'TransformedTransitionKernel',
 ]
 
-
```

Comment:
Please rebase on top of the latest master; this diff looks like it's undoing a change made in parallel.

Reply:
Done

Reply:
Fixed.